### PR TITLE
add api diff for between rc1 and rc2

### PR DESCRIPTION
**Pull request by Anipik**
cc @dotnet/ncl @tarekgh @tommcdon @krwq @ericstj 

@pranavkm @Tratcher @Pilchie @davidfowl

### CODE REVIEW

**Comment by tarekgh** [comment]
System.Globalization and Diagnostics looks good to me.

File: release-notes/5.0/preview/api-diff/rc2/.Net/5.0-rc2.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# API Difference net5.0-rc1 vs net5.0-rc2
+
+API listing follows standard diff formatting. Lines preceded by a '+' are
+additions and a '-' indicates removal.
+
+* [System.Diagnostics](5.0-rc2_System.Diagnostics.md)
+* [System.Globalization](5.0-rc2_System.Globalization.md)
```
Comment (Pilchie) [inline release-notes/5.0/preview/api-diff/rc2/.Net/5.0-rc2.md:7]: Pipelines isn't mentioned in the summary.

Comment (davidfowl) [reply release-notes/5.0/preview/api-diff/rc2/.Net/5.0-rc2.md:7]: We should remove the pipelines changes between RC1 and RC2 because its basically a wash.

Comment (ericstj) [reply release-notes/5.0/preview/api-diff/rc2/.Net/5.0-rc2.md:7]: It was removed from the shared framework, is that worth mentioning?

Comment (Anipik) [reply release-notes/5.0/preview/api-diff/rc2/.Net/5.0-rc2.md:7]: should we remove the addition in rc1 as well ?

**Review by pranavkm — APPROVED** [review]

**Comment by khellang** [comment]
@Anipik Will an api-diff be produced for netcoreapp3.1 vs net5.0 RTM?

**Comment by Anipik** [comment]
> @Anipik Will an api-diff be produced for netcoreapp3.1 vs net5.0 RTM?

yes its in progress

**Comment by Anipik** [comment]
https://github.com/dotnet/core/pull/5610

**Pull request merged**